react-components/src/tools/details/ToolInfoDialog.js

```diff
@@ -0,0 +1,227 @@
+                                                ? tool.container.memory_limit
+                                                : NOT_APPLICABLE}
+                                        </span>
+                                    </Grid>
+                                    <Grid item xs={12}>
+                                        <span className={labelClass}>
+                                            {getMessage("pidsLimitLabel")}:
+                                        </span>
+                                        <span className={valueClass}>
+                                            {tool.container.pids_limit
+                                                ? tool.container.pids_limit
+                                                : NOT_APPLICABLE}
+                                        </span>
+                                    </Grid>
+                                    <Grid item xs={12}>
+                                        <span className={labelClass}>
+                                            {getMessage("networkingLabel")}:
+                                        </span>
+                                        <span className={valueClass}>
+                                            {tool.container.network_mode
+                                                ? tool.container.network_mode
+                                                : NOT_APPLICABLE}
+                                        </span>
+                                    </Grid>
+                                    <Grid item xs={12}>
+                                        <span className={labelClass}>
+                                            {getMessage("secondsLimitLabel")}:
+                                        </span>
+                                        <span className={valueClass}>
+                                            {tool.time_limit_seconds
+                                                ? tool.time_limit_seconds
+                                                : NOT_APPLICABLE}
+                                        </span>
+                                    </Grid>
+                                </Grid>
+                            </Paper>
+                        )}
+                        {tabIndex === 1 && (
+                            <AppGridListing
+                                apps={apps}
+                                sortField="name"
+                                sortDir="asc"
+                                selectable={false}
+                                deletable={false}
+                                enableMenu={false}
+                                isSelected={() => false}
+                                selectedApps={[]}
+                                onSortChange={() => {}}
+                                resetAppSelection={() => {}}
+                                handleAppSelection={() => {}}
+                            />
+                        )}
```

Comment:
Not sure if there is a different way to avoid errors in console and at the same time doing nothing on certain callbacks.